src/api/code-challenge/content-types/code-challenge/lifecycles.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  beforeCreate(event) {
+    console.log("**** beforeCreate hook is activated! ****");
+    const [eventTests, eventMetaTests] = [
+      event.params.data.tests,
+      event.params.data.MetaTest,
+    ];
```

Comment:
This doesn't make sense. Take a second look at it

Reply:
You're right, but I am following the conventions of the Strapi Components. I could rename it to metaTests, but then it would look weird
<img width="198" alt="Screen Shot 2022-02-28 at 2 02 17 PM" src="https://user-images.githubusercontent.com/79945330/156079641-920f0130-8e92-48df-9ff3-85e90d71a28b.png">
 on strapi. Which one do you think we should prefer?